### DISCUSSION TITLE
MY FIRST TIME USING THIS IDK IF ANYONE SEES IT>

### DISCUSSION THREAD

**Discussion by emredapu** [openai/whisper#1347] [Q&A]
Hey guys. I'm trying to translate Dopa's educational League videos on youtube. After seeing some sentences translated awkwardly I thought, can we teach it the way players talk about the game so we can excavate the awkwardsness from the subs a little bit. Like it actually somewhat learns the game? Can anyone tell me that I'm deranged, or how  I can do it if it's possible? Thanks, this is amazing.

Edit 1: Just realized that it might be related to the translation part of the process. Is there a better way to translate Korean transcrypt specifying that it is a srt file for a League video? Or is it the transcrypt that needs attention since Whisper doesn't know LOL lingo?

**Comment by phineas-pta** [comment]
this has nothing to do with whisper, since your videos are very subject-specific, any translation services would struggle
